packages/admin-panel/package.json

```diff
@@ -28,6 +28,7 @@
     "url": "https://github.com/jacorreamed/booking-services/issues"
   },
   "dependencies": {
+    "@booking-services/shared": "^0.0.0",
```

Comment:
Since we change the version in the package.json of the shared package, we need to update this dependency in all the packages.

Just replace `@booking-services/shared@^0.0.0` by `@booking-services/shared@^0.0.1`